src/pages/AppList.svelte

```diff
@@ -26,16 +27,23 @@
   <input type="checkbox" bind:checked={showDeprecated} />
   Show deprecated applications
 </label>
+<label>
+  <input type="checkbox" bind:checked={showPrototype} />
+  Show prototype applications
+</label>
 {#if apps}
   <FilterInput onChangeText={filterApps} />
   {#each filteredApps as app}
-    {#if showDeprecated || !app.deprecated}
+    {#if (showDeprecated || !app.deprecated) && (showPrototype || !app.prototype)}
```

Comment:
What is the intended functionality here? If an app is both deprecated and a prototype and I only turn on the filter "Show prototype applications", should it show me this app?

Reply:
Welp, https://github.com/mozilla/glean-dictionary/pull/118#discussion_r505607807 this comment makes this much easier :)

Reply:
Yes, the intended functionality was to implement checkbox. But since there won't be much prototype apps I will go ahead and just show them directly at all times.